openfisca_core/parameters/helpers.py

```diff
@@ -2,17 +2,24 @@
 import traceback
 
 import numpy
+import pandas as pd
```

Comment:
If it is useful to test `pandas` here, I suggest we move this as an actual test, and that we require `pandas` optionally in case we want to run that specific test.